templates/shimexecutor-post-install-job.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}-post-install"
+  labels:
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      name: "{{ .Release.Name }}"
+      labels:
+        app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+        app.kubernetes.io/instance: {{ .Release.Name | quote }}
+        helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    spec:
+      restartPolicy: Never
+      serviceAccountName: {{ .Release.Name }}-post-install
+      containers:
+      - name: post-install-job
+        image: "bitnami/kubectl:1.30.0"
+        imagePullPolicy: IfNotPresent
+        command: ["/bin/sh", "-c"]
+        args:
+          - |-
+              cat <<EOF | kubectl apply -f -
+              apiVersion: core.spinoperator.dev/v1alpha1
+              kind: SpinAppExecutor
+              metadata:
+                name: containerd-shim-spin
+                namespace: default
```

Comment:
are we sure that we want the namespace to be "default"?

Reply:
That is the convention we've been following thus far... it seemed to make the most sense to create the default executor in the default namespace.